src/store/models/Person.js

```diff
@@ -40,8 +40,7 @@ Person.fields = {
     through: 'PersonSkillsToLearn',
     throughFields: [ 'person', 'skillToLearn' ]
   }),
-  postsTotal: attr(),
-  votesTotal: attr()
```

Comment:
snuck in?

Reply:
It did... I swear I stashed all the changes... This attr isn't used anywhere anywhoo